src/libUtils/IPConverter.cpp

```diff
@@ -29,9 +32,58 @@ const std::string IPConverter::ToStrFromNumericalIP(
   return std::string(str);
 }
 
-const boost::multiprecision::uint128_t IPConverter::ToNumericalIPFromStr(
-    const std::string& ipStr) {
-  struct in_addr ip_addr;
-  inet_pton(AF_INET, ipStr.c_str(), &ip_addr);
-  return (boost::multiprecision::uint128_t)ip_addr.s_addr;
+void LogBrand() {
+  cout << "Copyright (C) Zilliqa. Version 1.0 (Durian). "
```

Comment:
It will be changed to "Copyright (C) Zilliqa. Version 3.0 (Durian - Mao Shan Wang)." in PR https://github.com/Zilliqa/Zilliqa/pull/1057

Reply:
Done